pom.xml

```diff
@@ -65,6 +66,13 @@
             <version>${workflow.version}</version>
         </dependency>
 
+        <!-- Temporary - needed for the UserSelector describable -->
+        <dependency>
+            <groupId>com.synopsys.arc.jenkinsci.plugins</groupId>
+            <artifactId>job-restrictions</artifactId>
```

Comment:
If you depend on it, maybe makes sense to implement some restriction strategies based on this plugin later. Now there is an issue with Pipeline compatibility, so it won't work correctly.

Copy-pasting of the code and restricting it maybe an option.